src/pages/components/combobox.mdx

```diff
@@ -239,11 +239,63 @@ The Combobox component follows this structure:
   <Combobox>
     <Option />
     <Option />
+    <OptGroup>
+      <Option />
+    </OptGroup>
     {/* etc. */}
   </Combobox>
 </Field>
 ```
 
+When authoring a `Combobox`, prefer either a vanilla function and/or `Array.map` to generate options:
```

Comment:
Re-write to simplify and use [active voice](https://garden.zendesk.com/content/principles#use-the-active-voice). Something like:

```
Use `Array.map` to iterate over options.
```

Tools like https://hemingwayapp.com/ can help.